test/helpers/localNetAccounts.ts

```diff
@@ -126,64 +125,48 @@ export const LOCAL_NET_DEFAULT_ACCOUNTS: Account[] = [
   },
 ];
 
-async function processFile(
-  filePath: string,
-  processDataFunction: (data: string) => number,
-): Promise<number> {
-  try {
-    await access(filePath);
-  } catch {
-    await writeFile(filePath, "");
-  }
-
-  try {
-    const release = await lockfile.lock(filePath, {
-      retries: {
-        retries: 30,
-        minTimeout: 100,
-        maxTimeout: 500,
-      },
-    });
-
-    try {
-      const data = await readFile(filePath, "utf-8");
-      const processedData = processDataFunction(data);
-      await writeFile(filePath, processedData.toString());
-
-      return processedData;
-    } finally {
-      await release();
-    }
-  } catch (error) {
-    console.error("Error during working with a file:", error);
-    throw error;
-  }
-}
-
-function processData(data: string): number {
+/**
+ * Increases the index based on the given data from a file.
+ *
+ * @param {string} dataFromFile - The data obtained from a file.
+ * @throws {Error} If the data is not a valid number.
+ * @returns {string} The resulting index as a string.
+ */
+function increaseIndex(dataFromFile: string): string {
   let index: number;
 
-  if (data === "") {
+  if (dataFromFile === "") {
     index = 0;
   } else {
-    index = Number(data) + 1;
+    index = Number(dataFromFile) + 1;
```

Comment:
I think we still need some explanation here regarding `+ 1`

Reply:
I guess file stored actual index plus 1 ? Probably we can mention that somewhere

Reply:
Or why not to store index directly as it is?

Reply:
According the current logic, the file stores index that have already used by other process and for the new process we need to get the next one. 

I can add an explanation here, or change the logic somehow.

Reply:
small explanation is fine

Reply:
@akim-bow take a look please, seems now it's clear. I edited method docs and added a comment near the line.